coda/parser.py

```diff
@@ -90,7 +90,7 @@ def parse(self, value):
         """
         if not self.is_valid_coda(value):
             raise ValueError('The given value is not a valid coda content')
-        recordlist = unicode(value, 'windows-1252', 'strict').split('\n')
+        recordlist = value.decode('windows-1252', 'strict').split('\n')
```

Comment:
in python2 str is bytes and unicode is unicode
in python 3 str is unicode and bytes is bytes

since we are decoding here, the value param must be bytes (str in python 2), so you need to open the files in 'rb' mode (in tests and in parse_file)